README-FIPS.md

```diff
@@ -77,6 +81,78 @@ If you configured OpenSSL to be installed to a different location, the paths wil
 vary accordingly. In the rare case that you need to install the fipsmodule.cnf
 to non-standard location, you can execute the `openssl fipsinstall` command manually.
 
+Installing the FIPS provider and using it with the latest release
+=================================================================
+
+This normally requires you to download 2 copies of the OpenSSL source code.
+
+Download and build the (OpenSSL 3.0) FIPS provider
+--------------------------------------------------
+
+OpenSSL 3.0 is the only source that is currently FIPS validated.
```

Comment:
3.0.0 not 3.0.  Possibly also for the title.

Reply:
Surely it changes slowly enough that it can be updated in the source tree?

Reply:
But you can't change already released versions. So, lets say at some point in the future we release 3.0.15 and its not validated. The README-FIPS.md file will say that only 3.0.0 is validated (and maybe 3.0.8 by that point). Then later 3.0.15 gets validated. But its already released so you can't go back and amend the README-FIPS.md file in *that release*. So people will download it as the latest validated fips version. But the README-FIPS.md file will say that its not validated.

Reply:
Ah right, gotcha

Reply:
And yet, the actual instructions just a few lines down specify 3.0.0.

The other way around could be to be more vague here with regards to the version and refer to a web page on https://openssl.org where we have more exact details.